### PR TITLE
fix: `Uncaught TypeError` trying to access body from head tag in windows

### LINKED ISSUE #301: Bug:  Uncaught TypeError: Cannot read properties of null (reading 'classList') in windows
### URL(s)

https://eslint.org/play/

### What did you do?

1. On `windows`  OS
1. Open any browser and visit [playground](https://eslint.org/play/)
2. Open `console` in devtools

### What did you expect to happen?

- No errors in console
- Should add `win` to the body class, if site is opened on `windows` machine

### What actually happened?

Received `document.body` is null error in console

### Participation

- [X] I am willing to submit a pull request for this issue.

## Console
image
## Source
image

### Cause
Trying to access `document.body` from `head` tag https://github.com/eslint/eslint.org/blob/3fdd3964c4383b1f970f550aa887795a01bcc2d5/src/_includes/layouts/base.html#L128

### PR DESCRIPTION
**Pull request by shanpriyan**
Moved the windows `userAgent` check to body tag

Resolves #301

### CODE REVIEW

File: src/_includes/layouts/base.html [hunk 1]
```diff
@@ -163,6 +160,11 @@
     <script src="{{ '/assets/js/tabs.js' | url }}"></script>
     <script src="{{ '/assets/js/components-index.js' | url }}"></script>
     {%- endif -%}
+    <script>
```
Comment (harish-sethuraman) [inline src/_includes/layouts/base.html:163]: Just wondering why we'd need this piece of code? Are we applying any special style or using it somewhere? else this is totally un necessary?

Comment (shanpriyan) [reply src/_includes/layouts/base.html:163]: Seems like there is no special styling exists as of now and not able to find where this is being used also.

Comment (amareshsm) [reply src/_includes/layouts/base.html:163]: I guess we don't need this now. Previously this logic was added to set specific `font-family` for windows. After some time that styling code was removed.

Comment (shanpriyan) [reply src/_includes/layouts/base.html:163]: Shall I remove the userAgent check, then?

Comment (amareshsm) [reply src/_includes/layouts/base.html:163]: @kecrily Can you confirm this?

**Comment by amareshsm** [comment]
https://github.com/eslint/eslint.org/issues/301

File: src/_includes/layouts/base.html (hunk 1 above)
Comment (nzakas) [reply src/_includes/layouts/base.html:163]: This is needed to fix an issue with emoji flags not showing up in Chromium on Windows:
https://github.com/eslint/eslint.org/issues/183

**Review by snitin315 — APPROVED** [review]
LGTM

File: src/_includes/layouts/base.html (hunk 1 above)
Comment (harish-sethuraman) [reply src/_includes/layouts/base.html:163]: I don’t see that #213 makes use of the class win to show emojis? Or am I missing something 😅

Comment (kecrily) [reply src/_includes/layouts/base.html:163]: Yes. It was used to restrict twemoji to Windows and use the default emoji for other platforms. I was later told that twemoji should be used for all platforms. This code seems useless now, let's remove it.

Comment (kecrily) [reply src/_includes/layouts/base.html:163]: > I don’t see that #213 makes use of the class win to show emojis? Or am I missing something 😅

It's in #215

File: src/_includes/layouts/base.html [hunk 2]
```diff
@@ -163,6 +160,11 @@
...
+        if (/windows/i.test(navigator.userAgent)) {
+            document.body.classList.add('win')
+        }
+    </script>
```
Comment (kecrily) [inline src/_includes/layouts/base.html:167]:
```suggestion
```

File: src/_includes/layouts/base.html (hunk 1 above)
Comment (snitin315) [reply src/_includes/layouts/base.html:163]: Yes, if it's not used let's remove 👍🏻

**Comment by harish-sethuraman** [comment]
So previously we were using the class `.win` for applying emoji flags fonts for windows alone in this PR #215 but then this was reverted due to other issues but the win classlist addition code alone came back after the revert which is now not needed. So we can now consider removing the code as suggested by others as well :)

**Review by harish-sethuraman — APPROVED** [review]
LGTM Thanks ✨

**Comment by amareshsm** [comment]
In https://github.com/eslint/eslint.org/issues/183 PR emoji flag issue was fixed and a script was added to add the `win` class name for windows. Styles for that class were not added in the https://github.com/eslint/eslint.org/issues/183 PR.

later in this https://github.com/eslint/eslint.org/pull/215/ PR script block was moved inside the body and added styles. But that particular flags font family (`"Twemoji Country Flags"`) was only added to the `win` class name. It should include other existing text font families also. 

<table>
<tr>
<td> Styles added in  #215   </td> <td> Expected </td>
</tr>
<tr>
<td>

```css
.win {
    font-family: "Twemoji Country Flags" !important;
}
```

 </td>
<td>

```css
.win {
    font-family: "Inter",
            -apple-system,
            BlinkMacSystemFont,
            "Segoe UI",
            Roboto,
            Helvetica,
            Arial,
            sans-serif,
            "Apple Color Emoji",
            "Twemoji Country Flags",
            "Segoe UI Emoji",
            "Segoe UI Symbol";
}
```

</td>
</tr>
</table>

Since the other text font families were not added incorrect font was displayed in the language picker. So #215 was reverted by #222.

Just added this comment for context. If want to use the system's own flag on non-Windows systems and custom flags for windows systems then we need to add the script and styles. Otherwise, we can remove the script block which is currently not in use.

**Review by kecrily — APPROVED** [review]
LGTM

**Review by amareshsm — APPROVED** [review]

**Comment by amareshsm** [comment]
LGTM

**Review by nzakas — APPROVED** [review]

**Comment by nzakas** [comment]
Thanks for digging into that, everyone. For future reference, when there are multiple people agreeing, please feel free to merge the PR. You don’t need to wait for anyone on the TSC.

**Pull request merged**